src/components/StakingHierarchy.js

```diff
@@ -0,0 +1,465 @@
+    display: none;
+  }
+`
+
+const Gold = styled.span`
+  color: ${({ theme }) => theme.colors.stakingGold};
+  font-weight: 600;
+`
+
+const StakingHierarchy = () => {
+  return (
+    <Container>
+      <Section number="1">
+        <Ether>
+          <StyledEtherSvg size="100%" />
+        </Ether>
+        <Line />
+        <Header>
+          <h2>Solo home staking</h2>
+          <Pills className="subtext">
+            <p>
+              <em>Most impactful</em>
+            </p>
+            <p>Full control</p>
+            <p>Full rewards</p>
+            <p>Trustless</p>
+          </Pills>
+        </Header>
+        <Glyph>
+          <SoloGlyph />
+        </Glyph>
+        <Content>
+          <p>
+            Solo staking on the Ethereum Beacon Chain is the{" "}
+            <Gold>gold standard</Gold> for staking. It is trustless, provides
+            full participation rewards, and improves the decentralization of the
+            network.
+          </p>
+          <p>
+            Those considering solo staking should have at least 32 ETH, hardware
+            connected to the internet ~24/7, and some technical know-how.
+          </p>
+          <p>
+            Over time, tooling has improved to help users get past some of the
+            technical hurdles, including options to bypass the command-line
+            altogether, if that's not your thing.
+          </p>
+          <p>
+            Check out some solo staking tools, then get started staking on the
+            Prater Testnet over on the{" "}
+            <Link to="https://prater.launchpad.ethereum.org">
+              Staking Launchpad
+            </Link>
+            .
+          </p>
+          <ButtonLink
+            to="/staking/solo/"
+            onClick={() => {
+              trackCustomEvent({
+                eventCategory: `StakingHierarchy`,
+                eventAction: `Clicked`,
+                eventName: "clicked solo staking",
+              })
+            }}
+          >
+            More on solo staking
+          </ButtonLink>
+        </Content>
+      </Section>
+      <Section number="2">
+        <Ether>
+          <StyledEtherSvg size="90%" />
+        </Ether>
+        <Line />
+        <Header>
+          <h2>Staking as a service</h2>
+          <Pills className="subtext">
+            <p>Your 32 ETH</p>
+            <p>Your validator keys</p>
+            <p>Entrusted node operation</p>
+          </Pills>
+        </Header>
+        <Glyph>
+          <SaasGlyph />
+        </Glyph>
+        <Content>
+          <p>
+            Running your own hardware is always recommended if possible for the
+            strength and health of the network, but we recognize this isn't
+            always reasonable for all users.
+          </p>
+          <p>
+            If you don't want or don't feel comfortable dealing with hardware,
+            but still want to stake your 32 ETH, staking-as-a-service options
```

Comment:
Notice we use both "staking-as-a-service" & "staking as a service" in various places. Should be be consistent here? Or is it hyphenated when used as a noun?

I'm no grammar pro... defer to @minimalsm 😄

Reply:
Not a grammar pro either, but been trying to go with hyphenated when in the middle of a sentence so it's clear they're strung together... but in cases where it's obvious I tried to leave them out so it looks better. 

I'll second the deferal to @minimalsm  =)